.travis.yml

```diff
@@ -12,6 +12,11 @@ script:
     else
     npm test -- --browsers Firefox;
     fi
+after_success:
+  - npm run semantic-release
```

Comment:
this will do a release after every successful build even on feature branches i think, is that right? seems like we'd only want to auto-release after successful tests on master